app.js

```diff
@@ -4,16 +4,35 @@ const logger = require('morgan');
 const cookieParser = require('cookie-parser');
 const bodyParser = require('body-parser');
 const compression = require('compression');
-
 const session = require('express-session');
-
-// template stuff
+const methodOverride = require('method-override');
 const handlebars = require('handlebars');
 const layouts = require('handlebars-layouts');
 const handlebarsWax = require('handlebars-wax');
-const authHelper = require('./helpers/authentication');
 
 const app = express();
+
+// template stuff
+const authHelper = require('./helpers/authentication');
+
+// mung middleware
+const securityHeaders = require('./middleware/security_headers');
+const cors = require('./middleware/cors');
+
+// set custom response header for ha proxy
+if (process.env.KEEP_ALIVE) {
```

Comment:
move down, to habe app.use of requires above directly where it is used

Reply:
Done